src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/Models/Manifest.cs

```diff
@@ -30,6 +30,34 @@ public class Manifest
 
         [XmlAttribute(AttributeName = "Location")]
         public string Location { get; set; }
+
+        #region Properties to be used in new publishing flow
+
+        [XmlAttribute(AttributeName = "InitialAssetsLocation")]
+        public string InitialAssetsLocation { get; set; }
+
+        [XmlAttribute(AttributeName = "AzureDevOpsBuildId")]
+        public int? AzureDevOpsBuildId { get; set; }
+
+        [XmlAttribute(AttributeName = "AzureDevOpsBuildDefinitionId")]
+        public int? AzureDevOpsBuildDefinitionId { get; set; }
+
+        [XmlAttribute(AttributeName = "AzureDevOpsAccount")]
+        public string AzureDevOpsAccount { get; set; }
+
+        [XmlAttribute(AttributeName = "AzureDevOpsProject")]
+        public string AzureDevOpsProject { get; set; }
+
+        [XmlAttribute(AttributeName = "AzureDevOpsBuildNumber")]
+        public string AzureDevOpsBuildNumber { get; set; }
+
+        [XmlAttribute(AttributeName = "AzureDevOpsRepository")]
+        public string AzureDevOpsRepository { get; set; }
+
+        [XmlAttribute(AttributeName = "AzureDevOpsBranch")]
+        public string AzureDevOpsBranch { get; set; }
```

Comment:
In the manifest, is this going to be different than Branch?

Reply:
Ahh I think I see now? There are now both Github and AzDO versions of these properties, but Branch and Build and such were not removed from the manifest?